platform/net/routes_searcher_unix.go

```diff
@@ -20,10 +20,14 @@ func NewRoutesSearcher(runner boshsys.CmdRunner, _ InterfaceManager) RoutesSearc
 	return cmdRoutesSearcher{runner}
 }
 
-func parseRoute(ipString string) Route {
+func parseRoute(ipString string) (Route, error) {
 	var r = regexp.MustCompile(`(?P<destination>[a-z0-9.]+)(/[0-9]+)?( via (?P<gateway>[0-9.]+))? dev (?P<interfaceName>[a-z0-9]+)`)
 
 	match := r.FindStringSubmatch(ipString)
+	// Issue #230, skip blackhole routes
```

Comment:
could you please remove the comment? the issue will already be tied via the PR/commit.